<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow1</class>
 <widget class="QMainWindow" name="MainWindow1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>319</width>
    <height>279</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>309</width>
      <height>253</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>309</width>
       <height>253</height>
      </rect>
     </property>
     <widget class="QLineEdit" name="lineEdit_pas">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>190</y>
        <width>133</width>
        <height>20</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>160</y>
        <width>84</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe Script</family>
        <pointsize>12</pointsize>
        <weight>9</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">
font: 75 12pt &quot;Segoe Script&quot;;
color:rgb(14, 255, 14);</string>
      </property>
      <property name="text">
       <string>Username</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_login">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>160</y>
        <width>133</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>190</y>
        <width>82</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe Script</family>
        <pointsize>12</pointsize>
        <weight>9</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">
font: 75 12pt &quot;Segoe Script&quot;;
color:rgb(14, 255, 14);</string>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>246</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>-10</y>
        <width>321</width>
        <height>271</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Вихід</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_registation">
      <property name="geometry">
       <rect>
        <x>231</x>
        <y>190</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="text">
       <string>Registration</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_login">
      <property name="geometry">
       <rect>
        <x>231</x>
        <y>161</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>311</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 11pt &quot;Verdana&quot;;
/*font: 75 8pt &quot;MS Shell Dlg 2&quot;;*/
color: rgb(131, 255, 129);
/*font: 75 8pt &quot;Georgia&quot;;*/</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <zorder>label_5</zorder>
     <zorder>lineEdit_pas</zorder>
     <zorder>lineEdit_login</zorder>
     <zorder>label_3</zorder>
     <zorder>label</zorder>
     <zorder>label_2</zorder>
     <zorder>pushButton</zorder>
     <zorder>pushButton_registation</zorder>
     <zorder>pushButton_login</zorder>
     <zorder>label_4</zorder>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>319</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
